### ISSUE TITLE
Add JScrollPane to phylogeny view

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lonelyjoeparker/qmul-genome-convergence-pipeline#39]
```
Add scroll pane behaviour to phylogeny view (or scaling)  - large phylogenies 
are drawn off the edge of the viewport.


```

Original issue reported on code.google.com by `joeparke...@gmail.com` on 13 Feb 2015 at 11:27

**Comment by lonelyjoeparker** [comment]
https://github.com/lonelyjoeparker/qmul-genome-convergence-pipeline/commit/2b72a87df37d8b0aa699c98a6f077903190a5a81 addresses this partially, but phylogeny and tip labels are still way off. Also noticed new issue #44, some trees have too many tip labels (or too few branches)...

**Comment by lonelyjoeparker** [comment]
Tip labels placement addressed by https://github.com/lonelyjoeparker/qmul-genome-convergence-pipeline/commit/3d8deb778419cb1d6bb59336df8a23f77ffae9f5

**Comment by lonelyjoeparker** [comment]
This issue now closed though NB #44 outstanding from general PhylogenyDisplayPanel implementation.
